rpcFirewall/dllmain.cpp

```diff
@@ -655,9 +655,9 @@ void waitForFurtherInstructions()
 
 	if (uninstallEvent != nullptr)
 	{
-		HANDLE allEvents[2];
-		allEvents[0] = uninstallEvent;
-		allEvents[1] = configurationUpdatedEvent;
+		// HANDLE allEvents[2];
```

Comment:
The configurationUpdatedEvent is obsolete and was replaced by a periodic check.
Please delete the unused code, as well as 
```//DWORD dwWaitResult = WaitForMultipleObjects(2, allEvents, false, INFINITE);```
Also please delete uninstallEvent entirely.

Generally deleting unused code would be appreciated and is preferred over commenting it out. If we find that it should have been used, we shall correct it.
Kindly apply to rest of commit by deleting instead of uncommenting (and change the commit message).